app/assets/javascripts/discourse/app/adapters/rest.js

```diff
@@ -9,7 +9,7 @@ const ADMIN_MODELS = [
   "embeddable-host",
   "web-hook",
   "web-hook-event",
-  "flagged-topic",
+  "flagged-topic"
```

Comment:
I don't think we need any of the formatting changes in this file.

Reply:
I think these were generated by prettier ... running eslint reverted them.